lua/lazyvim/plugins/extras/ui/alpha.lua

```diff
@@ -23,11 +23,11 @@ return {
       dashboard.section.header.val = vim.split(logo, "\n")
       -- stylua: ignore
       dashboard.section.buttons.val = {
-        dashboard.button("f", " " .. " Find file",       "<cmd> Telescope find_files <cr>"),
-        dashboard.button("n", " " .. " New file",        "<cmd> ene <BAR> startinsert <cr>"),
-        dashboard.button("r", " " .. " Recent files",    "<cmd> Telescope oldfiles <cr>"),
-        dashboard.button("g", " " .. " Find text",       "<cmd> Telescope live_grep <cr>"),
-        dashboard.button("c", " " .. " Config",          "<cmd> lua require('lazyvim.util').telescope.config_files()() <cr>"),
+        dashboard.button("f", " " .. " Find file",       LazyVim.pick() --[[@as string]]),
```

Comment:
Can you remove these @as string annotations? They're functions, even if dashboard.button would complain, I'd rather have correct dagnostics

Reply:
Sorry about that. I'm the one who suggested this. Just so I know for future reference, I should just ignore such LSP diagnostics?

Reply:
It's probably because the types of dashboard.button are incorrect, so can only be fixed there.
but of course doesn't really matter as long as we know it's correct

Reply:
@folke I've gone ahead and removed the type annotations in the most recent commit.